pkg/server/plugin/upstreamauthority/gcpcas/gcpcas_test.go

```diff
@@ -0,0 +1,58 @@
+package gcpcas
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/spiffe/spire/proto/spire/common/plugin"
+	"github.com/spiffe/spire/test/util"
+	"github.com/stretchr/testify/require"
+	// 	"github.com/spiffe/spire/pkg/server/plugin/upstreamauthority"
+)
+
+func TestGCPCAS(t *testing.T) {
+	ctx := context.Background()
+	fmt.Println("Hi")
```

Comment:
I realize this was probably just left in, but in general we shouldn't be outputting stuff from the `fmt` package in tests and should prefer `t.Logf` instead.